survey_csv_formatter.py

```diff
@@ -0,0 +1,114 @@
+    backer_dict['backer_details']['zip_code'] = row["Shipping Postal Code"]
+    backer_dict['backer_details']['phone'] = row["Shipping Phone Number"]
+
+    backer_dict['product_skus']['AEY-PROP-01'] = (
+       int(row["Everything in previous tier"]) + int(row["ARCANA PROPHETIA"])
+    )
+    backer_dict['product_skus']['AEY-PROP-02'] = (
+       int(row["Everything in previous tier"]) + int(row["Additional Metal Sigils"])
+    )
+    backer_dict['product_skus']['AEY-PROP-03'] = (
+       int(row["Everything in previous tier"]) + int(row["Physical Art and Lore Zine"])
+    )
+    backer_dict['product_skus']['AEY-KAWA-01'] = int(row["KAWA: A Game About Flow"])
+
+    return temp_dict
+
+# Creation of two dictionaries of backer data, one for responders, other for non-responders  
+responder_dict = {}
+non_response_dict = {}    
+for row in ks_response:
+  if is_survey_responder(row):
+    create_backer_entry(row)
+    responder_dict[int(row["Backer UID"])] = create_backer_entry(row)
+  else:
+    non_response_dict[int(row["Backer UID"])] = create_backer_entry(row)
+
+# Function to convert above dictionaries into lists for csv writing
+# Each SKU ordered by each is a new item in list (so backer with SKU A and B is 2 items)    
+def create_csv_list(target_dict):
+    temp_list = []
+
+    for backer in target_dict:
+        for sku in target_dict[backer]['product_skus']:
+            if target_dict[backer]['product_skus'][sku] > 0:
+
+                temp_list.append([])
+                csv_row = temp_list[(len(temp_list))-1]
+
+                csv_row.append("HZ02")
+                csv_row.append(backer)
+                csv_row.append(target_dict[backer]['backer_details']['first_name'])
+                csv_row.append(target_dict[backer]['backer_details']['last_name'])
+                csv_row.append(target_dict[backer]['backer_details']['address_line_1'])
+                csv_row.append(target_dict[backer]['backer_details']['address_line_2'])
+                csv_row.append(target_dict[backer]['backer_details']['city'])
+                csv_row.append(target_dict[backer]['backer_details']['state'])
+                csv_row.append(target_dict[backer]['backer_details']['country_code'])
+                csv_row.append(target_dict[backer]['backer_details']['zip_code'])
+                csv_row.append(target_dict[backer]['backer_details']['phone'])
+                csv_row.append(sku)
+                csv_row.append(target_dict[backer]['product_skus'][sku])
+                csv_row.append("by Becky")
+                csv_row.append(target_dict[backer]['backer_details']['email'])
+                csv_row.append(target_dict[backer]['remarks'])
+
+    return temp_list
+
+# Creation of list for responders and non-responders
+shipping_list = create_csv_list(responder_dict)
+non_response_list = create_csv_list(non_response_dict)
+
+# Function to write responder and non-responder lists into csvs
```

Comment:
Nit: Reading the code it's clear it's a function, no need to write "Function to".

See example docstring
https://www.programiz.com/python-programming/docstrings

Reply:
Same as above

Reply:
https://github.com/AutumnVulpes/survey-csv-formatter/compare/e955525bde90dc05f99bc403de48217d719a13d7..de50cfc25f91a3f2070dcce442c25e01dfd9d326